docs/README.rst

```diff
@@ -100,6 +100,12 @@ General
 |                       |         |          |                                  | to verify the BIG-IP                    |                |
 |                       |         |          |                                  | configuration.                          |                |
 +-----------------------+---------+----------+----------------------------------+-----------------------------------------+----------------+
+| vs-snat-pool-name     | string  | Optional | n/a                              | Name of the SNAT pool that all virtual  |                |
```

Comment:
Ya Booch said the they only care about automap or snat, since I guess it is difficult to set up the lists that are used when none is configured so we are leaving it out.